File: fixtures/python/syntax_error.py
def valid_function(x: int) -> int:
    """A valid function."""
    return x + 1


def another_valid(s: str) -> str:
    return s.upper()


# This has a syntax error
def broken(x: int ->:
    return x


def after_error(y: int) -> int:
    return y * 2

File: fixtures/rust/syntax_error.rs
pub fn valid_function(x: i32) -> i32 {
    x + 1
}

pub fn another_valid(s: &str) -> String {
    s.to_uppercase()
}

// This line has a syntax error
pub fn broken(x: i32 -> {
    x
}

pub fn after_error(y: i32) -> i32 {
    y * 2
}
